shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -1594,4 +1597,11 @@ object GlutenConfig {
       .doc("Log granularity of AWS C++ SDK in velox.")
       .stringConf
       .createWithDefault("FATAL")
+
+  val VELOX_ORC_SCAN_ENABLED =
+    buildStaticConf("spark.gluten.sql.columnar.backend.velox.orc.scan.enabled")
+      .internal()
+      .doc(" Enable velox orc scan. If disabled, vanilla spark orc scan will be used.")
+      .booleanConf
+      .createWithDefault(false)
```

Comment:
can we set default to true? ORC format is already used by some companies.

Reply:
Sure, updated. I thought we would want to keep it off until the data correctness is resolved.